problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -0,0 +1,157 @@
+function MentoringWithStepsBlock(runtime, element) {
+
+    var steps = runtime.children(element).filter(
+        function(c) { return c.element.className.indexOf('pb-mentoring-step') > -1; }
+    );
+    var step = $('.mentoring', element).data('step');
```

Comment:
@itsjeyd nit: technically, it's not a step, it's step number. Consider renaming to `step_number`